docs/src/misc/about.md

```diff
@@ -17,6 +17,7 @@
 - [Nick Dewaele](https://github.com/Nikdwal) contributed the [Tucker manifold](../manifolds/tucker.md)
 - [Renée Dornig](https://github.com/r-dornig) contributed the [centered  matrices](../manifolds/centeredmatrices.md) and the [essential manifold](../manifolds/essentialmanifold.md)
 - [David Hong](https://github.com/dahong67) contributed uniform distributions on the Stiefel and Grassmann manifolds.
+- [Simon Jacobsson](https://github.com/sjacobsson) contributed the [Segre manifold](../manifolds/segre.md) including its [warped metric](@ref segre-warped-metric-sec) thereon.
```

Comment:
We started (actually after you started this PR) to list all contributions beyond fixing typos and that are not done by the main developers on this page.

I added you with your GitHub page as link. Feel free to change this to any link you would like to link this to.
I can also add you to the .zenodo file if you like – we do not write “and contributors” there but list everyone who is listed here as well in the metadata of the zenodo DOI entry

Reply:
Thanks!

Sure, you can add me to the zenodo file if you want. I have an orcid that I try to keep up to date
https://orcid.org/0000-0002-1181-972X